### DISCUSSION TITLE
Prometheus: Lengthy exemplar modals

### DISCUSSION THREAD

**Discussion by gtk-grafana** [grafana/grafana#61715] [Observability]
Some users have so many labels, and the exemplar modal can take up the entire vertical real estate of the viewport (or more, causing the panels to resize as the vertical scroll bar is added/removed).

Can we provide a way for users to specify which labels are shown in the exemplar modal?

@grafana/observability-metrics

**Comment by itsmylife** [comment]
I believe we can give a way to configure which fields will be displayed in the modal via transformations.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82810. Feel free to continue the discussion around this there. Thank you!
